plugins/outputs/cratedb/README.md

```diff
@@ -0,0 +1,38 @@
+# CrateDB Output Plugin for Telegraf
+
+This plugin writes to [CrateDB](https://crate.io/) via its [PostgreSQL protocol](https://crate.io/docs/crate/reference/protocols/postgres.html).
+
+## Table Schema
+
+The plugin requires a a table with the following schema.
```

Comment:
Extra `a` in this sentence

Reply:
https://github.com/influxdata/telegraf/pull/3210/commits/6c2df9e9c765cea1a8631e12dcd828b40f58b467